### PR TITLE
Onboarding update

### PR DESCRIPTION
**Pull request by curtisbarnard**
## Proposed changes

- Add a markdown file to the docs folder with the onboarding information to make it more rapidly accessible than going to the google doc at: https://docs.google.com/document/d/1MJMMOdlzyBOVlvKHydSQLTaxcbNqkS1NQufxiJK9jDs/edit#
- Update the broken link on README to point to the new markdown file.

There is a lot of good info in the doc that would be great for first-timers, but also isn't necessary to show on the main README

### CODE REVIEW

**Comment by curtisbarnard** [comment]
Not sure what the failed test is about. I only touched the README and created a new markdown in the docs folder.

**Comment by sidetrackedmind** [comment]
Thanks for these changes @curtisbarnard. I will approve and we can work on the assignment list at a later date.

File: docs/onboarding.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+4. Join our virtual meeting on Thursdays at 7pm, with call information shared via the Slack channel.
+
+## Our development process
+
+1. [Create a new GitHub branch.](https://github.com/codeforpdx/opentransit-metrics)
+2. Build your feature.
+3. Make a pull request. Request reviews from members of the team; you’ll need their approval to merge.
+
+- Good people to ask for reviews:
```
Comment (sidetrackedmind) [inline docs/onboarding.md:29]: We'll try to do a comprehensive update to this with new team members in another PR.

**Review by sidetrackedmind — APPROVED** [review]
Looks great - thanks for making these changes. I think newly on-boarded folks will find them very helpful.

**Pull request merged**